src/api/spec/models/binary_release_spec.rb

```diff
@@ -85,7 +85,12 @@
 
       subject { described_class.update_binary_releases_via_json(repository, [repeated_binary_hash]) }
 
-      it { expect { subject }.to change(BinaryRelease, :count).by(-1) }
+      it 'sets the obsolete_time on the duplicated binary release record' do
```

Comment:
@adrianschroeter can you confirm that this change in behavior is intended?

Reply:
I am not sure which data set get removed. Is there a way to see the datasets of the repository and see which one disappears?